packages/cli/medusa-cli/src/create-cli.ts

```diff
@@ -422,42 +362,6 @@ function buildLocalCommands(cli, isLocalProject) {
         })
       ),
     })
-    .command({
```

Comment:
todo: as discussed this should be added back for users self hosting on a single machine to benefit from their resources while increasing capabilities 👍

Reply:
Suggestion to group the commands:

`medusa server:dev`
`medusa server:start`
`medusa server:start --cluster`

Or just:

`medusa server:start --dev --cluster`

Reply:
Done. But did not move the commands to a namespace as that might require further discussions